src/s1reader/s1_annotation.py

```diff
@@ -73,11 +73,11 @@ def element_to_dict(elem_in: ET, dict_tree: dict = None):
         text_elem = elem_in.text
 
         if key_elem in dict_datatype_rfi:
-            value_castor = dict_datatype_rfi[key_elem]
+            elem_datatype = dict_datatype_rfi[key_elem]
         else:
-            warnings.warn(f'Datetype Castor for {key_elem} was not defined. Considering the value as string.')
-            value_castor = str
-        dict_tree[key_elem] = value_castor(text_elem)
+            warnings.warn(f'Data type for element {key_elem} is not defined. Casting the value "{text_elem}" as string.')
```

Comment:
Thank you, @seongsujeong . Could you also split this line so it's not too long?

Reply:
Absolutely. I've split the warning message into two lines.